CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## Changelog
 
+#### WIP
+- Added support for passing ecto `dynamic` filters
+- Added support for passing functions to use for custom filtering
+- Added `EctoShorts.filter` as a shortcut to `CommonFilters.convert_params_to_filter` 
+- Added a default order_by field and direction for queries
```

Comment:
Default ordering shouldn't exist, this can cause performance issues, also the direction has a default naturally, the order param was actually removed because of this